style.css

```diff
@@ -250,3 +650,349 @@ body {
   display: inline-block;
   width: 20px;
 }
+
+/* Recent Works section */
+
+.recent-work {
+  background-color: white-background-color;
+  padding: 80px 0;
+  position: relative;
+}
+
+.about-me {
+  position: relative;
+  top: 50px;
+}
+
+.recent-work-container {
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+}
+
+.section-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.recent-work h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  align-items: center;
+  margin-bottom: 40px;
+  color: headline-color;
+}
+
+.title-container {
+  position: relative;
+}
+
+.section-indicator {
+  display: inline-block;
+  width: 48px;
+  height: 8px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  bottom: -3px;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 30px;
+}
+
+.card-grid {
+  display: grid;
+  grid-template-areas:
+    "card-img card-img card-img"
+    "card-title card-title card-title"
+    "tags tags tags"
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please stick to the Figma design your divider is not centred but in the Figma design it's centred so kindly fix it :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-30 04-14-32](https://user-images.githubusercontent.com/45226848/235328124-c41e48e7-c94b-4388-b44f-fbd98713e442.png) | ![Screenshot from 2023-04-30 04-24-21](https://user-images.githubusercontent.com/45226848/235328128-0711a504-c37f-4749-bfa8-40e181615ab0.png) |